packages/react/src/reactrouterv6-compat-utils.tsx

```diff
@@ -427,10 +427,10 @@ function getNormalizedName(
         // If path is not a wildcard and has no child routes, append the path
         if (path && !pathIsWildcardAndHasChildren(path, branch)) {
           const newPath = path[0] === '/' || pathBuilder[pathBuilder.length - 1] === '/' ? path : `/${path}`;
-          pathBuilder += newPath;
```

Comment:
Originally I thought to change these lines to
```js
const newPath = prefixWithSlash(path);
pathBuilder = trimSlash(pathBuilder) + newPath;
```
but `newPath` is used in a different path concatenation further down the method and, even though no tests fail, I didn't want collateral damage. (Tests probably _should_ have failed though - I think the path concatenation in this method could be better handled, and is under-tested.)